integration/integration.go

```diff
@@ -100,7 +100,7 @@ func fixOrder(t testing.TB, doc *wirebson.Document) {
 	}
 }
 
-// fixActualUpdateN replaces the int64 `nMatched`, `nModified`, `nUpserted`, `n` fields with a int32 values.
+// fixActualUpdateN replaces the int64 `nMatched`, `nModified`, `nUpserted`, `n` fields with int32 values.
```

Comment:
The default correction would have been `an`... `int32`, but that doesn't make sense in the context of `values` (plural).